get_dependencies.py

```diff
@@ -24,11 +24,22 @@ def download(url, filename):
     sys.stdout.write('\n')
```

Comment:
Don't change the permissions of this file from 100644 to 100755.

Reply:
i haven't made any change here.  may you expain it more?

Reply:
You have changed the permissions of the file. You can use `chmod 0644 FILE_NAME` to change them back to 0644.

Reply:
okay